pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -30,6 +32,8 @@ type Config struct {
 	Logger                 logger.Logger
 	MaxTuplesPerWriteField int
 	MaxTypesPerModelField  int
+	// for serializing the read changes token
+	TokenSerializer encoder.ContinuationTokenSerializer
```

Comment:
The fact that you had to leave this comment reinforces my opinion here: https://github.com/openfga/openfga/pull/2020/files#r1824830100

Reply:
Out of date in #2064